### PR TITLE
{chem}[foss/2019b] TINKER v8.7.2

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Depends on: https://github.com/easybuilders/easybuild-easyblocks/pull/2102

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/478903211dd6ba708f4e21bb45d92442 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/57ae467d57b3bb135069e929bfeab13f for a full test report.

File: easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+source_urls = ['https://dasher.wustl.edu/tinker/downloads/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(name)s-%(version)s_fix_BAR_mode_1.patch']
+checksums = [
+    'f9e94ae0684d527cd2772a4a7a05c41864ce6246f1194f6c1c402a94598151c2',  # tinker-8.7.2.tar.gz
+    'c5122fe5ed778ceacf9bad144c5eb381506cab407a7733a6de05b4252bd1e9cf',  # TINKER-8.7.2_fix_BAR_mode_1.patch
+]
+
+runtest = False
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb:19]: why the change to skipping tests?

Comment (akesandgren) [reply easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb:19]: As far as I remember one of the tests simply never managed to finish. It just ran for hours and hours and ...

Comment (migueldiascosta) [reply easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb:19]: can you add a comment about that?

File: easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name = 'TINKER'
+version = '8.7.2'
+
+homepage = 'https://dasher.wustl.edu/tinker'
+description = """The TINKER molecular modeling software is a complete and general package for molecular mechanics
+ and dynamics, with some special features for biopolymers."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'openmp': True}
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb:9]: doesn't seem to be necessary, the tests are using threads even if built without this (and even this is set to `False`, so it can actually be misleading?)

**Comment by akesandgren** [comment]
Looks like the code doesn't even use FFTW unless we patch it to activate it.

**Comment by akesandgren** [comment]
Fix for this coming up as a change to the tinker.py easyblock..., i.e. skipping tests that takes too long.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/9a433c7645338aaa7ebab8619069ed98 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/84834d9133e13ea9543d4e91a268c7ad for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node369.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b493372391d86fca23a9ca5ebed8a98c for a full test report.

File: easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb:9]: @akesandgren can you address this comment?

Comment (akesandgren) [reply easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb]: Yeah, I dropped it.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node368.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ba233683b6c4b1e8c0e9b2d263784d9d for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @akesandgren !
